src/components/auth/_auth.scss

```diff
@@ -0,0 +1 @@
+@import './loginForm';
```

Comment:
This willl be where all .scss files are imported from this auth component directory